pkgs/by-name/po/poetry/plugins/poetry-plugin-lambda-build.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  docker,
+  poetry-core,
+  pytest-cov,
+  pytestCheckHook,
+  poetry,
+  pip,
+}:
+
+buildPythonPackage rec {
+  pname = "poetry-plugin-lambda-build";
+  version = "2.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "micmurawski";
+    repo = "poetry-plugin-lambda-build";
+    tag = version;
+    hash = "sha256-i9vFLiuEXTsNhdvSPQQUh2EYwSjlmgRepplf24eddeA=";
+  };
+
+  build-system = [ poetry-core ];
+
+  dependencies = [
+    docker
+  ];
+
+  nativeCheckInputs = [
+    pytest-cov
+    pytestCheckHook
+    poetry
+    pip
+  ];
+
+  preCheck = ''
+    export POETRY_CACHE_DIR=$TMPDIR/.cache/pypoetry
+    export POETRY_VIRTUALENVS_PATH=$TMPDIR/.cache/pypoetry/virtualenvs
+  '';
+
+  # it's kinda stupid to include this here, but maybe upstream would be open to
+  # including it.
```

Comment:
Why is it stupid? I don't understand the point of this comment.
Did you open an upstream issue?
Why not set `autouse=False` (and add a comment with the reason) if you want to disable the fixture?
Anyway, this seems entirely unnecessary.